### ISSUE TITLE
The app attempts to switch but my PG32UCDM won't display anything.

### ISSUE THREAD

**Issue by SaccoSVD** [res2k/HDRTray#10]
I have a PG32UCDM and a M28U. The M28U is HDR capable on paper although is useless so I never use HDR on it.

I can toggle HDR in the PG32UCDM using the Asus app.

With HDRTray what happens is that it toggles HDR on (The Windows Display dialog HDR toggle button reports it on) but the PG32UCDM doesn't display anything. Doesn't seem to finish the screen mode change.

When that happens I just toggle the HDR button in the Windows Display dialog to go back to SDR.

So, in short. I cannot use HDRTray in my system.

**Comment by res2k** [comment]
Hm. This is unfortunate, but I'm not sure how much I can do about it right now. But maybe we can collect some information on the issue.

FYI, internally, HDRTray just uses Windows API functions to switch display(s) to HDR mode - so, in theory, it should work as well (or not) as the HDR toggle in the Windows Display settings.

That said, does that toggle work correctly?
Does switching HDR via the game bar hotkey (Win+Alt+B) work correctly?

Lastly, with some superficial googling, I found reports relating to your monitor and HDR, but I'm not sure how relevant they are (https://www.reddit.com/r/OLED_Gaming/comments/1d4dmxi/pg32ucdm_cant_switch_between_different_hdr_types/). Perhaps see if you can find other reports of HDR-related issues that are similar to yours, maybe they can give some hints.

**Comment by SaccoSVD** [comment]
I think the problem is that HDRTray attempts to turn HDR globally so both my monitors go HDR mode.

I guess HDR is a kind of a mess and Windows is not making turning HDR on a specific monitor that easily.

**Comment by res2k** [comment]
> I guess HDR is a kind of a mess and Windows is not making turning HDR on a specific monitor that easily.

Technically, it's possible to enable it per display - but support for that is currently not implemented. See #7.
